integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/DeleteExecution.scala

```diff
@@ -55,52 +55,98 @@ import org.apache.carbondata.spark.DeleteDelataResultImpl
 object DeleteExecution {
   val LOGGER = LogServiceFactory.getLogService(this.getClass.getName)
 
+  def deleteDeltaExecution(
```

Comment:
add comment to describe its functionality

Reply:
it is just a refactor to move the long method to short.